docs/Administration.rst

```diff
@@ -2479,7 +2479,7 @@ This Spaces menu contains two choices:
 
 
 
-.. note:: the list of templates is read-only. Changing a template's details requires to edit the config file as explained here.
+.. note:: the list of templates is read-only. Changing a template details requires to edit the config file as explained in :ref:`developers guide <PLFDevGuide.SpaceTemplates.Extend>` .
```

Comment:
@plamarque link of the developer guide added.

Reply:
Thanks but in fact I'm not sure the note is at the right place. There is another note below inside the Manage Telpaltes section that seems better place than this one. i would remove this pone and change the onther one to insert the link.